docs/.vuepress/tools/jsdoc-convert/jsdoc.js

```diff
@@ -20,6 +20,13 @@ const whitelist = [
   'editors/baseEditor/baseEditor.js',
   '3rdparty/walkontable/src/cell/coords.js',
   'plugins/copyPaste/focusableElement.js',
+  'DataMap.js',
+  'translations/maps/hidingMap.js',
+  'translations/maps/indexesSequence.js',
+  'translations/maps/trimmingMap.js',
+  'utils/samplesGenerator.js',
+  'translations/maps/physicalIndexToValueMap.js',
+  'utils/ghostTable.js'
```

Comment:
With these newly generated files, I'm concern that the search takes the content of that files and shows in the results. Most of these files are private and they are only referred from somewhere in the code. IMO they should not be part of the search result

e.g. `DataMap` or `GhostTable` are hidden from the public API.

![image](https://user-images.githubusercontent.com/571316/122410066-e84da480-cf83-11eb-9078-e3693ecc5a7a.png)

This can be done in a separate issue, if at all.

Reply:
Please create a new issue for that